ecosystem_tests/ecosystem_tests_cli/ecosystem_tests.py

```diff
@@ -138,7 +138,15 @@ def __init__(self):
         a developer sees an option. It it can receive arguments, it's a
         method - if not, it's an attribute.
         """
-
+        self.version = click.option('-V',
+                                    '--version',
+                                    type=click.STRING,
+                                    help=helptexts.VERSION)
```

Comment:
Also does it mean we need to provide the plugin version every time we run the command?